app/players/player.py

```diff
@@ -0,0 +1,14 @@
+from abc import abstractmethod, ABC
+
+
+class Player(ABC):
+    def __init__(self, nickname: str) -> None:
+        self.nickname = nickname
+
+    @abstractmethod
+    def get_rating(self) -> None:
+        pass
```

Comment:
Leaving None is misleading here. You can put here expected types that will be returned in child instances, or `Any` type in case if different childs can have different return types in same methods.